### DISCUSSION TITLE
Add tkinter library

### DISCUSSION THREAD

**Discussion by AZachia** [Akascape/tkinter-toolkit#10] [Add this]
Hello,

I have made 2 tkinter library, and I would like to add them in your project.
The first one  allows to create rounded windows and widgets (experimental for now) for windows.
The second one is based on the first one and allows to create widgets in the first meaning of the word (to display information from the windows desktop). It provides a collection of premade widgets, like a clock, a volume and luminosity control...

<img width="959" alt="screenshot" src="https://github.com/Akascape/tkinter-toolkit/assets/154826103/e7341011-cdc9-459e-88a0-a36441318290">
<img width="274" alt="RoundedTk" src="https://github.com/Akascape/tkinter-toolkit/assets/154826103/cdd10381-dddc-499b-a848-d5f25db9db31">

I let you the links of the 2 libraries:
- [Rounded-TK](https://github.com/AZachia/Rounded-Tk)
- [Windows-Widgets](https://github.com/AZachia/Windows-Widgets)

I hope you will can add them in your project, and I stay at your disposal for any question.

**Comment by Akascape** [comment]
@AZachia 
I added the rounded-tk, but let me know if it works in all the platforms, including linux? 
Because you can see this issue here: https://stackoverflow.com/questions/75594687/how-to-apply-transparent-background-in-tkinter-window-of-linux-not-alpha
